scheduler/server/priority_based_scheduling_alg.go

```diff
@@ -1,255 +0,0 @@
-package server
```

Comment:
If this is deleted, why do we still have comments that reference the priority based algo?

Reply:
fixed comments